docs/instruments/bmi.json

```diff
@@ -0,0 +1,532 @@
+        "options": {
+          "order": [
+            "height_inches",
+            "height_inches_status"
+
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupWeightLbs": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your weight (lbs) :"
+        },
+        "options": {
+          "order": [
+            "weight_lbs",
+            "weight_lbs_status"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupHeightCms": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your height (cms) :"
+        },
+        "options": {
+          "order": [
+            "height_cms",
+            "height_cms_status"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupWeightKgs": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your weight (kgs) :"
+        },
+        "options": {
+          "order": [
+            "weight_kgs",
+            "weight_kgs_status"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      }
+    },
+    "setup": [
+      {
+        "name": "topPage",
+        "type": "page",
+        "description": {
+          "en-CA": "Top"
+        },
+        "order": [
+          "noteUnits",
+          "unit_classification",
+          "sectionStandard",
+          "sectionMetric",
+          "bmi",
+          "bmi_category"
+        ]
+      }
+    ]
+  },
+  "ui": {
+    "noteUnits": {
+      "type": "static"
+    },
+    "unit_classification": {
+      "type": "select",
+      "options": {
+        "showEmptyOption" : true
+      }
+    },
+    "height_feet": {
+      "type": "number"
+    },
+    "height_feet_status": {
+      "type": "select",
+      "options": {
+        "showEmptyOption" : true
+      }
+    },
+    "height_inches": {
+      "type": "number"
+    },
+    "height_inches_status": {
+      "type": "select",
+      "options": {
+        "showEmptyOption" : true
+      }
+    },
+    "weight_lbs": {
+      "type": "number"
+    },
+    "weight_lbs_status": {
+      "type": "select",
+      "options": {
+        "showEmptyOption" : true
+      }
+    },
+    "height_cms": {
+      "type": "number"
+    },
+    "height_cms_status": {
+      "type": "select",
+      "options": {
+        "showEmptyOption" : true
+      }
+    },
+    "weight_kgs": {
+      "type": "number"
+    },
+    "weight_kgs_status": {
+      "type": "select",
+      "options": {
+        "showEmptyOption" : true
+      }
+    },
+    "bmi": {
```

Comment:
If this is the Score field, the scoring calculation is found in the *.score file (for a given linst) : 

https://github.com/aces/Loris/blob/minor/docs/instruments/bmi.score